stack/zod/utilizacao.md

```diff
@@ -0,0 +1,158 @@
+---
+order: 1
+icon: rocket
+label: "Utilização"
+author:
+    name: Matheus das Neves Fernandes
+date: 2024-03-15
+category: Utilização
+tags:
+    - data validation
+    - utilização
+---
+
+# Utilização 
+
+Aqui vamos mostrar o importante e essencial para poder usufruir-se de uma biblioteca para validação dos seus dados em algum projeto. Para maior aprofundamento e uma validação mais complexa, sugiro que consulte a [documentação do Zod](https://zod.dev).
+
+## Zod Schema
+
+Assim como montamos nosso tipo no typescript, no zod iremos criar um Zod Type. Porém, existem infinitas possibilidades para estruturar seu schema de forma bem simples.
+ Imagine que queremos montar um Zod Schema que representa o seguinte tipo:
+
+```ts
+import { z } from 'zod'
```

Comment:
Esse `import` é necessário aqui? Se não for, melhor tirar. Acho que pode confundir, fazendo pensar que isso já é código usando zod.

Reply:
Ah, é pra mostrar que precisa importar o zod antes de usar alguma coisa

Reply:
Eu deixo sempre no início pra mostrar que é necessário importar, antes do código em si